scripts/aks/teardown.sh

```diff
@@ -282,21 +275,21 @@ az aks delete \
     --only-show-errors
 echo "Done."
 
-
 #######################################################################################
 ### Delete Redis Cache
 ###
 
 WORKDIR_PATH="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-echo "Deleting Redis Cache for QA..."
+echo ""
+printf "%s► Execute Redis Cache for QA %s%s\n" "${grn}" "$WORKDIR_PATH/../delete_redis_cache_for_console.sh" "${normal}"
```

Comment:
inconsistent use of printf/echo

```suggestion
printf "\n%s► Execute Redis Cache for QA %s%s\n" "${grn}" "$WORKDIR_PATH/../delete_redis_cache_for_console.sh" "${normal}"
```